tests/Makefile

```diff
@@ -48,51 +52,75 @@ load-images: ## Load NGF and NGINX images on configured kind cluster
 load-images-with-plus: ## Load NGF and NGINX Plus images on configured kind cluster
 	cd .. && make PREFIX=$(PREFIX) TAG=$(TAG) load-images-with-plus
 
-test: ## Run the system tests against your default k8s cluster
-	go test -v ./suite $(GINKGO_FLAGS) -args --gateway-api-version=$(GW_API_VERSION) \
-		--gateway-api-prev-version=$(GW_API_PREV_VERSION) --image-tag=$(TAG) --version-under-test=$(NGF_VERSION) \
-		--plus-enabled=$(PLUS_ENABLED) --ngf-image-repo=$(PREFIX) --nginx-image-repo=$(NGINX_PREFIX) \
-		--pull-policy=$(PULL_POLICY) --k8s-version=$(K8S_VERSION) --service-type=$(GW_SERVICE_TYPE) \
-		--is-gke-internal-lb=$(GW_SVC_GKE_INTERNAL)
+.PHONY: setup-gcp-and-run-tests
+setup-gcp-and-run-tests: create-gke-router create-and-setup-vm run-tests-on-vm ## Create and setup a GKE router and GCP VM for tests and run the functional tests
 
-.PHONY: delete-kind-cluster
-delete-kind-cluster: ## Delete kind cluster
-	kind delete cluster
+.PHONY: setup-gcp-and-run-nfr-tests
+setup-gcp-and-run-nfr-tests: create-gke-router create-and-setup-vm nfr-test ## Create and setup a GKE router and GCP VM for tests and run the NFR tests
 
-.PHONY: run-tests-on-vm
-run-tests-on-vm: ## Run the tests on a GCP VM
-	bash scripts/run-tests-gcp-vm.sh
+.PHONY: create-gke-cluster
+create-gke-cluster: ## Create a GKE cluster
+	bash scripts/create-gke-cluster.sh $(CI)
 
 .PHONY: create-and-setup-vm
 create-and-setup-vm: ## Create and setup a GCP VM for tests
 	bash scripts/create-and-setup-gcp-vm.sh
 
-.PHONY: cleanup-vm
-cleanup-vm: ## Delete the test GCP VM and delete the firewall rule
-	bash scripts/cleanup-vm.sh
-
 .PHONY: create-gke-router
 create-gke-router: ## Create a GKE router to allow egress traffic from private nodes (allows for external image pulls)
 	bash scripts/create-gke-router.sh
 
-.PHONY: cleanup-router
-cleanup-router: ## Delete the GKE router
-	bash scripts/cleanup-router.sh
+.PHONY: sync-files-to-vm
+sync-files-to-vm: ## Syncs your local NGF files with the NGF repo on the VM
+	bash scripts/sync-files-to-vm.sh
 
-.PHONY: setup-gcp-and-run-tests
-setup-gcp-and-run-tests: create-gke-router create-and-setup-vm run-tests-on-vm ## Create and setup a GKE router and GCP VM for tests and run the tests
+.PHONY: run-tests-on-vm
+run-tests-on-vm: ## Run the functional tests on a GCP VM
+	bash scripts/run-tests-gcp-vm.sh
+
+.PHONY: nfr-test
+nfr-test: ## Run the NFR tests on a GCP VM
+	bash scripts/run-tests-gcp-vm.sh true
```

Comment:
would it make sense to introduce a variable here rather than a `true` argument.  It makes it hard to see what `true` means here.

Reply:
>  I can change back to an env var if that's preferred.

👍  I'd prefer make it more readable